mshadow/half.h

```diff
@@ -10,7 +10,11 @@
 #include "./base.h"
 
 #if MSHADOW_USE_F16C
-  #include <x86intrin.h>
+  #if defined(_MSC_VER)
+    #include <intrin.h>
```

Comment:
You had mentioned in your PR description that msvc doesn't have intrin.h. If so why would we want to include this? Especially given that below _cvtsh_ss is turned off for MSVC?

Reply:
I wrote it wrong. It's only "intrin.h", no "x86intrin.h""